apps/dashboard/src/components/Plans/PlanForm.tsx

```diff
@@ -46,7 +46,8 @@ const PlanForm: React.FC<{
   close: Void;
   refresh: Void;
   plan?: IPlan.MappedAttributes;
-}> = ({ open, close, refresh, plan }) => {
+  setOpen?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
I don't understand why do we have to assign this complex type to `setOpen`. 
```suggestion
  setOpen?: (open: boolean) => void;
```